quasar.conf.js

```diff
@@ -35,7 +35,7 @@ module.exports = function() {
     build: {
       vueRouterMode: "hash", // available values: 'hash', 'history'
       // transpile: false,
-
+      env: require("./src/boot/env-parser"),
```

Comment:
We don't need to use .env any longer, since we have a proxy configuration with `/api/` prefix in both deployment and development environments. Please review PR #35